api/lib/opentelemetry/correlation_context/propagation/context_keys.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module CorrelationContext
+    module Propagation
+      # The ContextKeys module contains the keys used to store correlations
+      # in a {Context} instance
+      module ContextKeys
+        extend self
+
+        CORRELATION_CONTEXT_KEY = 'correlation-context'
+        private_constant(:CORRELATION_CONTEXT_KEY)
+
+        def span_context_key
```

Comment:
I find it slightly confusing to call this `span_context_key` when it is actually the correlation context key.